server/routes/createPlan/hopingToGetWork/hopingToGetWorkController.test.ts

```diff
@@ -243,7 +245,6 @@ describe('HopingToGetWorkController', () => {
     })
 
     it('On success - UPDATE - hopingToGetWork change to POSITIVE - Sets session record then redirects to qualifications', async () => {
-      config.featureToggles.useNewInductionApiEnabled = true
```

Comment:
Resetting the `mockInductionService` revealed the fact that the service wasn't being called here! It took me a while to work out why, but it's because this test is checking the switch from the short route to the long route.